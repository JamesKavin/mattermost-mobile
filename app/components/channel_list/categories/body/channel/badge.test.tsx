// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
// See LICENSE.txt for license information.

import React from 'react';

import {renderWithIntlAndTheme} from '@test/intl-test-helper';

import Badge from './badge';

describe('components/channel_list/categories/body/channel/badge', () => {
    it('should match snapshot', () => {
        const wrapper = renderWithIntlAndTheme(
            <Badge
                count={10}
                muted={false}
            />,
        );
        expect(wrapper.toJSON()).toMatchSnapshot();
    });
});
