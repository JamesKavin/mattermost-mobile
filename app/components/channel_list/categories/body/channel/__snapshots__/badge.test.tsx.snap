// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/channel_list/categories/body/channel/badge should match snapshot 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "#ffffff",
        "borderRadius": 10,
        "height": 20,
        "justifyContent": "center",
        "marginTop": 5,
        "paddingHorizontal": 8,
        "textAlignVertical": "center",
      },
      false,
    ]
  }
>
  <Text
    style={
      Array [
        Object {
          "color": "#1e325c",
          "fontFamily": "OpenSans",
          "fontSize": 12,
          "fontWeight": "400",
          "lineHeight": 16,
        },
        false,
      ]
    }
  >
    10
  </Text>
</View>
`;
